### DISCUSSION TITLE
Mispost: NavigationPolygon doesn't have hints for .get_vertices() in GD4.0b2

### DISCUSSION THREAD

**Discussion by Black-Virtue** [godotengine/godot-proposals#5541] [Scripting]
As it says.
Type hints for most of the other properties and methods show fine.
Using the method works fine, just doesn't hint.

**Comment by Zireael07** [comment]
Not a place for bug reports

**Reply by Black-Virtue** [reply]
Apologies, must have clicked the wrong submission.
